src/convexhull.jl

```diff
@@ -0,0 +1,31 @@
+"""
+    upperhull(pset)
+
+Upper part of the convex hull of a given set of 2D points `pset`. 
+The upperhull is computed according to the Andrew's Monotone Chain Algorithm.
+See https://en.wikipedia.org/wiki/Convex_hull_algorithms.
+
+The input is sorted on x-coordinate, and the hull is computed in two steps producing the upper and the lower parts of the hull respectively. 
+This is a modification of the Graham's Scan algorithm, devised by A. M. Andrew, known as Andrew's Monotone Chain Algorithm. It has the same basic properties as Graham's scan. 
```

Comment:
Long lines such as this should be split into multiple lines. Try to follow the number of columns of the previous paragraph.